Makefile

```diff
@@ -63,10 +63,9 @@ endif
 
 #### Tools ####
 
-ifeq ($(shell type mips-linux-gnu-ld >/dev/null 2>/dev/null; echo $$?), 0)
-  MIPS_BINUTILS_PREFIX := mips-linux-gnu-
-else
-  $(error Please install or build mips-linux-gnu)
+MIPS_BINUTILS_PREFIX ?= mips-linux-gnu-
```

Comment:
Could you move this to the `Defaults` section? Probably also a good idea to add the comment from OoT's makefile as well.